### DISCUSSION TITLE
Strange behaviour (from a newbee's view

### DISCUSSION THREAD

**Discussion by GerhardKreuzer** [marcokreeft87/room-card#326] [General]
Hi,
I tried some configs. I get the icon instead of the value and if I want to supress the name I get the name ...
I am very unsure, is it my fault or is there a lot of bugs?
Please help me out.

Thanks a lot.

Gerhard

![Strange behaviour Screenshot 2024-04-09 104907](https://github.com/marcokreeft87/room-card/assets/4210407/9569a181-d813-410f-8874-36217606804b)

![image](https://github.com/marcokreeft87/room-card/assets/4210407/d38b75cd-4c54-48bd-8edc-14f55f67783e)

**Comment by marcokreeft87** [comment]
Well it helps if you read the wiki:
https://github.com/marcokreeft87/room-card/wiki/Configuration

There aren't bugs with what you are trying to do
